doc/reference/networking/net_hostname.rst

```diff
@@ -6,6 +6,26 @@ Hostname Configuration
 Overview
 ********
 
+A networked device might need a hostname in certain cases like if the device
+is configured to be a mDNS responder (see :ref:`dns_resolve_interface` for
+details) and needs to respond to ``<hostname>.local`` DNS queries.
+
+The :option:`CONFIG_NET_HOSTNAME_ENABLE` must be set in config file in order
```

Comment:
set in which config file?  or should we just delete "in config file"

Reply:
I will delete the "in config file" part, it is useless.